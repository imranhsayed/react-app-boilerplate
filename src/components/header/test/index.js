/**
 * Test for header component.
 *
 * @package react-app-boilerplate
 */

/**
 * External dependencies
 */
import renderer from 'react-test-renderer';

/**
 * Internal dependencies
 */
import Header from '../index';

describe( 'Test Header Component', () => {
	it( 'Should match Header snapshot and render correctly', () => {
		const component = renderer.create( <Header /> );

		const tree = component.toJSON();

		expect( tree ).toMatchSnapshot();
	} );
} );
