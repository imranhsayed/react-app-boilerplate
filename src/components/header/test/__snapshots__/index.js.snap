// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Header Component Should match Header snapshot and render correctly 1`] = `
<h1>
  Header
</h1>
`;
